### PR TITLE
feat(styling): add more default styles and update storybook examples

### PR DESCRIPTION
**Pull request by marcoeh**
Default styles in action:
- https://deploy-preview-41--lucid-nuxt-2021.netlify.app/en/test-page
- https://storybook--lucid-nuxt-2021.netlify.app/?path=/story/richtext--html-kitchen-sink

Slightly inspired by https://newcss.net/

### CODE REVIEW

File: assets/scss/abstracts/_variables.scss [hunk 1]
```diff
@@ -25,6 +26,9 @@ $s-line-height-std: 1.3;
 $s-font-weight-std: 400;
 $s-font-weight-bold: 700;
 
+/* Border Radius */
+$s-border-radius-default: 4px;
```
Comment (FabianEllenberger) [inline assets/scss/abstracts/_variables.scss:30]: [we could also use `px2rem` here](https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius)

File: assets/scss/base/_reset.scss [hunk 2]
```diff
@@ -2,50 +2,69 @@
...
-th,
-ul {
-  font-weight: normal;
-  margin: 0;
-  padding: 0;
+textarea,
+ul,
+video {
+  /* Margins for most elements */
+  margin-bottom: 1rem;
```
Comment (FabianEllenberger) [inline assets/scss/base/_reset.scss:44]: i could imagine this being a little bit annoying having to remove an element from this list each time building something and wanting to have a clean (no margin) element, what do you think @marcoeh ?

File: assets/scss/base/_reset.scss [hunk 3]
```diff
@@ -60,3 +79,8 @@ input {
 address {
   font-style: normal;
 }
+
+textarea {
+  /* Don't let the <textarea> extend off the screen naturally or when dragged by the user */
```
Comment (FabianEllenberger) [inline assets/scss/base/_reset.scss:84]: like those details 👍

File: assets/scss/base/_reset.scss (hunk 2 above)
Comment (marcoeh) [reply assets/scss/base/_reset.scss:44]: I don't think this is an issue and spacings should be something that requires attention anyway in every project. So having some default margin-bottom makes sense and for new projects it has to be adjusted globally or on a component level.
image

Comment (marcoeh) [reply assets/scss/base/_reset.scss:44]: Scoped version:
image

**Pull request merged**